openreview/venue_request/venue_request.py

```diff
@@ -907,6 +907,16 @@ def setup_request_form(self):
                 'required': False,
                 'order': 9
             },
+            'secondary_area_chairs': {
+                'description': 'Are you going to have secondary area chairs?.',
```

Comment:
```suggestion
                'description': 'Does your venue have Secondary Area Chairs?',
```

Just in case we ever decide to unhide this